### PR TITLE
XWIKI-21106: Main content area in view mode still has borders when th…

### PR DESCRIPTION
**Pull request by lucaa**
…e body background and the page content background are of the same color

* Removed box-shadow from the page content container in view mode
* Also removed the box-shadow in edit mode (which fixes also XWIKI-21108)

### CODE REVIEW

**Comment by Sereza7** [comment]
For the sake of referencing on Jira (following [best practice](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRule:AlwaysputaJIRAissuereferenceincommitmessages)):
https://jira.xwiki.org/browse/XWIKI-21106 
and https://jira.xwiki.org/browse/XWIKI-21108

**Review by Sereza7 — APPROVED** [review]
For the sake of referencing on Jira (following [best practice](https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HRule:AlwaysputaJIRAissuereferenceincommitmessages)):
https://jira.xwiki.org/browse/XWIKI-21106

LGTM :+1:

**Review by manuelleduc — APPROVED** [review]
+1 to apply the fix on master. -1 to apply on LTS branches.

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/layout.less [hunk 1]
```diff
@@ -302,6 +303,23 @@
...
+
+/* Protect against nesting of the breadcrumb inside #mainContentArea, e.g., in AdminSheet and other containers that
+already take care of padding and shadow. */
+#mainContentArea #hierarchy_breadcrumb, #xwikieditor #hierarchy_breadcrumb {
+  padding-bottom: @line-height-computed;
+  /* Padding and shadow is already in the #mainContentArea element */
+  padding-top: 0;
+  box-shadow: none;
+}
+
```
Comment (lucaa) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/layout.less:322]: @manuelleduc I must admit that I don't understand how the changes from this commit are related to ticket XWIKI-21106, maybe an additional comment on the ticket itself with examples of what else needs to be handled besides what I described could help here...

Comment (manuelleduc) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/layout.less:322]: I made a temporary mistake while fixing a merge conflict. This is not part of the final commit, sorry.

**Pull request merged**